builder/deploy/deployer.go

```diff
@@ -631,14 +636,16 @@ func (d *deployer) startAccountingRequestMeter(resMap map[string]string, svcRes
 	}
 }
 
-func getValidSceneType(scene int) types.SceneType {
-	switch scene {
-	case 0:
+func getValidSceneType(deployType int) types.SceneType {
+	switch deployType {
+	case types.SpaceType:
 		return types.SceneSpace
-	case 1:
+	case types.InferenceType:
 		return types.SceneModelInference
-	case 2:
+	case types.FinetuneType:
 		return types.SceneModelFinetune
+	case types.ServerlessType:
+		return types.SceneModelInference
```

Comment:
should return types.ServerlessType?

Reply:
both serverless and inference deploy type  mapping to scene-inference for accounting.